bindings/python/tests/bindings/test_pre_tokenizers.py

```diff
@@ -119,3 +119,29 @@ def test_instantiate(self):
         assert isinstance(Digits(True), Digits)
         assert isinstance(Digits(False), Digits)
         assert isinstance(pickle.loads(pickle.dumps(Digits())), Digits)
+
+
+class TestCustomPreTokenizer:
+    class BadCustomPretok:
+        def pre_tokenize(self, pretok, wrong):
+            pass
+
+    class GoodCustomPretok:
+        def split(self, n, normalized):
+            return [normalized, normalized]
+
+        def pre_tokenize(self, pretok):
+            pretok.split(self.split)
+
+    def test_instantiate(self):
+        bad = PreTokenizer.custom(TestCustomPreTokenizer.BadCustomPretok())
+        good = PreTokenizer.custom(TestCustomPreTokenizer.GoodCustomPretok())
+
+        assert isinstance(bad, PreTokenizer)
+        assert isinstance(good, PreTokenizer)
+        with pytest.raises(Exception, match="TypeError: pre_tokenize()"):
+            bad.pre_tokenize_str("Hey there!")
+        assert good.pre_tokenize_str("Hey there!") == [
```

Comment:
I think a more realistic example would be better, here I find it a bit confusing, and it would be hard to change this test into creating my own CustomPretok.